examples/seismic/model.py

```diff
@@ -375,9 +375,9 @@ def update(self, name, value):
             param = getattr(self, name)
         except AttributeError:
             # No physical parameter with tha name, create it
-            setattr(self, name, self._gen_phys_param(name, value, self.space_order))
+            setattr(self, name, self._gen_phys_param(value, name, self.space_order))
```

Comment:
We need to put this under CI somehow

Reply:
> I overall don't have an issue with this as it adds test/example cases but I think it should be done as a proper constant tti as currently it just slightly modifies the current VP constant only

I tried to fix a mistake (constant case was not running) without changing the test design. The test only update the `vp` parameter, so I left the things as they were.

Reply:
All that comes into my mind is 
``` 
from examples.seismic import demo_model
import numpy as np


def test_model_updt(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0),
                    space_order=4, nbl=10):

    # Layered model (tti)
    tti_model = demo_model('layers-tti', shape=shape, spacing=spacing,
                           space_order=space_order, nbl=nbl)
    tpl1_set = set(tti_model.physical_parameters)

    # Layered model (isotropic)
    iso_model = demo_model('layers-isotropic', shape=shape, spacing=spacing,
                           space_order=space_order, nbl=nbl)
    tpl2_set = set(iso_model.physical_parameters)

    # Physical parameters in either set but not in the intersection.
    diff_phys_par = tuple(tpl1_set ^ tpl2_set)

    # Convert iso model in tti model
    for i in diff_phys_par:
        iso_model.update(i, getattr(tti_model, i).data[nbl:-nbl, nbl:-nbl, nbl:-nbl])
        assert np.array_equal(getattr(iso_model, i).data, getattr(tti_model, i).data)
```
I can add this script to the `tests` folder if you agree